packages/shared/src/lib/featureManagement.ts

```diff
@@ -1,5 +1,10 @@
 import { IFlags } from 'flagsmith';
 
+export enum AdditionalInteractionButtons {
```

Comment:
We should probably follow this for all future features.
What do you guys think?

Reply:
Yep, I like it, maybe it even makes sense to move types to a separate file, so it's not going to grow as fast as one file will be + if easier to scroll through it